### PR TITLE
fix: create symbolic link on request to server

### LINKED ISSUE #256: The git diff return error while using in values repo
**Is your feature request related to a problem? Please describe.** A clear and concise description of what the problem
I'm always frustrated when I get an error using `git diff` in values repo.
```
git diff
error loading config: no matching creation rules found
fatal: unable to read files to diff
```
Ability to inspect the changes critical for reviewing changes before committing them.

Running `otomi bootstrap` also does not solve the issue.

**Describe the solution you'd like** A clear and concise description of what you want to happen.
Configure git for values repo while performing  `otomi bootstrap`

**Describe alternatives you've considered** A clear and concise description of any alternative solutions or features
Provide some help information.

**Additional context** Add any other context or screenshots about the feature request here.

### PR DESCRIPTION
**Pull request by j-zimnowoda**
fixes #256

### CODE REVIEW

File: src/server/index.ts [hunk 1]
```diff
@@ -15,12 +15,19 @@ export const stopServer = (): void => {
   server?.close()
 }
 
+function symlinkEnvDir() {
+  const repoPath = '/tmp/otomi-values'
+  const envPath = 'env'
+  if (!existsSync(repoPath)) throw new Error(`The git repository does not exist at ${repoPath} path`)
```
Comment (Morriz) [inline src/server/index.ts:21]: This is never the case as we designed it to be there. No need for defending, and expecting it to "may be" different.

File: src/server/index.ts [hunk 2]
```diff
@@ -15,12 +15,19 @@ export const stopServer = (): void => {
...
+  if (!existsSync(envPath)) symlinkSync(repoPath, envPath)
```
Comment (Morriz) [inline src/server/index.ts:22]: Don't do if here. It _must_ break if its not there.

**Review by Morriz — CHANGES_REQUESTED** [review]
Hope this fixes it. Minor remarks related to dev principles. Like, don't defend against errors (those must surface and be fixed)

**Comment by j-zimnowoda** [comment]
It is not defending but postponed check.

**Comment by Dunky13** [comment]
Could it be that the server is started before a symlink is created?
https://github.com/redkubes/otomi-core/pull/564/files#diff-e4e7a68d82cdfba4e7dab9a32a8f55f3806ffa0cf33f1b8ab64b42941cf68888L61-L63

**Review by Morriz — APPROVED** [review]

**Comment by Morriz** [comment]
great...tnx

**Review by Morriz — APPROVED** [review]

**Comment by Dunky13** [comment]
1 more change coming - to use ENV_DIR

**Review by Dunky13 — APPROVED** [review]

**Pull request merged**